fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -81,6 +106,13 @@ load_env() {
     source "${script_dir}/${run_env_sh}"
   fi
 
+  if [ -z "${MAX_CORES}" ]; then
```

Comment:
Agreed, but maybe its good as a sign to indicate its origin. Otherwise it is a a bit too unspecific.